libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/SourceViewEditText.java

```diff
@@ -43,18 +41,4 @@ public boolean onKeyPreIme(int keyCode, KeyEvent event) {
     public void setOnImeBackListener(OnImeBackListener listener) {
         this.mOnImeBackListener = listener;
     }
-
-    private void setCustomTypeface(AttributeSet attrs) {
```

Comment:
Unrelated to this PR, should this extend `AppCompatEditText` rather than `EditText`?